book.toml

```diff
@@ -82,3 +82,22 @@ line-numbers = true
 
 [output.exerciser]
 output-directory = "comprehensive-rust-exercises"
+
+[output.i18n]
+default_language = "en"
+translate_all_languages = true
+optional = true
+
+[output.i18n.languages]
+"en" = "English"
+"es" = "Spanish (Español)"
+"ko" = "Korean (한국어)"
+"pt-BR" = "Brazilian Portuguese (Português do Brasil)"
+
+[output.tera-backend]
+after = "html"
+components = [
+    { name = "Macros", path = "src/components/macros.html" },
```

Comment:
I put them in `src` but `theme` would probably be better

Reply:
I agree: `mdbook` considers `src/` it's own directory and it will for example copy files it finds there to the destination directory.